### ISSUE TITLE
Training loss drops very slowly

### ISSUE THREAD

**Issue by lyj157175** [wenet-e2e/wenet#540]
Hello, I used the provided pre-trained model to test, and the effect is the same as in the readme. But when I use aishell-1 to train the u2++conformer model on my own single GPU, the parameter speed_perturb is False and grad_clip is 5, and the other parameters seem to be unchanged. In the 10-40 rounds of epoch, the loss has been around 60, and the drop is very slow. And when I tested the epoch40 model, I found that it was basically not recognized on the test set. Is this normal, so I can train to 360 epochs? Where did I go wrong

**Comment by lyj157175** [comment]
May I ask how long aishell-1 has been trained with 8 GPUs. How long and how many rounds do I probably train for a GPU?

**Comment by muntasir2000** [comment]
I have the same question and facing same issue. I'm training with ~200 hours of data (u2++). After 200 epoch, my loss is around 22. I have restarted the training 2 times increasing the learning rate too. The good thing is loss is still decreasing, but very slowly. And with a RTX 3090, I'm using batch size 24 which is occupying 22GB of total 24GB GPU memory. But the GPU utilization is very low, at max it reaches 50% and on average 30-40%. 

How many epoches should I train for?

**Comment by lyj157175** [comment]
Can we add a WeChat to chat?
my  wechat id:lyj1571754229

**Comment by robin1001** [comment]
please talk in

**Comment by JaySpeech** [comment]
I have the same issue,do you find the solution?
